new-client/src/plugins/GeosuiteExport/GeosuiteExportModel.js

```diff
@@ -670,6 +673,61 @@ class GeosuiteExportModel {
       return layer.id === id && layer.type === "vector";
     });
   };
+
+  zipDocuments = (documents) => {
+    let zip = new JSZip();
+    let deliveryInfo = "";
+    let usedFileNames = [];
+
+    documents.forEach((doc) => {
+      if (doc.selected) {
+        let fileLink = doc.link;
```

Comment:
Same as above. There are probably more `let` declarations that can be changed to `const` if you want to :) However, i will accept the pr without you changing them. If you update the import of jsZip i am happy!